app/models/user.rb

```diff
@@ -19,9 +19,8 @@ def authorized_to?(access_request)
 
   def inventory_groups
     # No need to fetch inventory groups if the RBAC feature is globally disabled
-    return Rbac::ANY if rbac_disabled?
-    # No groups should be available when using certificate authentication
-    return [] if cert_authenticated?
+    # All groups should be available when using certificate authentication
+    return Rbac::ANY if rbac_disabled? || cert_authenticated?
```

Comment:
Authenticated requests with certificate would have access to everything regardless of group restrictions. Is is possible to tie the certificate to a specific system?

Reply:
Valid point, found some stuff to read in the inventory README about this...will experiment a bit.

Reply:
@vkrizan fixed